Orange/widgets/visualize/owvenndiagram.py

```diff
@@ -139,7 +139,6 @@ def setData(self, data, key=None):
             # Store hints only on the first setData call.
             self._storeHints()
             self._inputUpdate = True
-
```

Comment:
I think here is [another whitespace change](https://github.com/biolab/orange3/pull/1070/files?diff=unified#r55426507) that has nothing to do with this PR.